src/tests/examples.rs

```diff
@@ -562,3 +562,21 @@ fn test_generic_for_loop(#[case] backend: BackendKind) -> miette::Result<()> {
 
     Ok(())
 }
+
+#[rstest]
+#[case::kimchi_vesta(BackendKind::KimchiVesta(KimchiVesta::new(false)))]
+#[case::r1cs(BackendKind::R1csBls12_381(R1CS::new()))]
+fn test_generic_builtin_bits(#[case] backend: BackendKind) -> miette::Result<()> {
+    let public_inputs = r#"{"xx":"2"}"#;
+    let private_inputs = r#"{}"#;
```

Comment:
I'm thinking more and more that we should have the inputs to these tests closer to each files. Maybe we could start our files with the inputs as comments? For example `generic_builtin_bits.no`:

```rust
// public_inputs = {"xx":"2"}
// private_inputs = {}
fn main(_) { ... }
```